tests/test-solr.php

```diff
@@ -177,4 +177,88 @@ public function test_index_site() {
         $this->assertEquals(10, $resultSet->getNumFound());
         $this->assertEquals(0, $indexResult);
     }
+
```

Comment:
Please run PHPCS on this and autofix/fix errors. Too many to put here.

Reply:
yeah, don't think it really matters if we meet the standards for tests.

Reply:
ok